mmv1/api/resource.go

```diff
@@ -1976,3 +1976,13 @@ func (r Resource) CodeHeader(templatePath string) string {
 func (r Resource) MarkdownHeader(templatePath string) string {
 	return strings.Replace(r.CodeHeader(templatePath), "//", "#", -1)
 }
+
+// ====================
+// TGC
+// ====================
+// Converts most properties during cai2hcl, exept for computed properties
```

Comment:
```suggestion
// Filters out computed properties during cai2hcl
```
Doesn't seem to do any converting on its own, just filtering?